app/classes/Transvision/ShowResults.php

```diff
@@ -461,7 +487,13 @@ public static function resultsTable($search_object, $search_results, $page)
                       <a class='bug_link' target='_blank' href='{$bz_link[0]}'>
                         &lt;report a bug&gt;
                       </a>
-                      {$meta_target}
+                      {$meta_target}";
+            if ($locale2 == 'sr' && !$extra_locale) {
+                $table .= "<input type='button' value='To latin' data-transliterated-id='{$string_id}' class='transliterate_button'>";
+            }
+            $table .= "
+                      <span class='clipboard' data-clipboard-target='#{$regular_string_id}' alt='Copy to clipboard'><img src='/img/copy_icon_black_18x18.png'></span>
```

Comment:
This should be removed, it’s already here https://github.com/mozfr/transvision/pull/754/files#diff-4d976c6b5c0ea357b6a48453ca776e91R405